test/Altinn.Platform.Authentication.Tests/Controllers/RequestControllerTests.cs

```diff
@@ -695,12 +698,88 @@ public async Task Get_All_Requests_By_System_Paginated_Several()
         Assert.Contains(list, x => x.PartyOrgNo == "910493353");
         Assert.NotNull(res2.Links.Next);
 
-        HttpResponseMessage message3 = await client.GetAsync(res2.Links.Next);
+        HttpResponseMessage message3 = await client2.GetAsync(res2.Links.Next);
         Assert.Equal(HttpStatusCode.OK, message3.StatusCode);
         Paginated<RequestSystemResponse>? res3 = await message3.Content.ReadFromJsonAsync<Paginated<RequestSystemResponse>>();
         Assert.True(res3 is not null);
     }
 
+    [Fact]
+    public async Task Delete_Request_ByGuid_Accepted()
```

Comment:
may be can add a test for requestid not found

Reply:
will do in future housekeeping PR